frontend/app/src/views/history/ledger-actions/LedgerActionContent.vue

```diff
@@ -347,7 +347,7 @@ export default defineComponent({
     const confirmationMessage: Ref<string> = ref('');
     const expanded: Ref<LedgerActionEntry[]> = ref([]);
     const valid: Ref<boolean> = ref(false);
-    const form = ref<LedgerActionForm | null>(null);
+    const form = ref<any>(null);
```

Comment:
Do you think we can get this _(InstanceType)_ working? https://vuejs.org/api/general.html#definecomponent